README.rst

```diff
@@ -37,6 +37,32 @@ Create an Enum-class and pass it as first argument to the Django model EnumField
     class Beer(models.Model):
         style = enum.EnumField(BeerStyle, default=BeerStyle.LAGER)
 
+You can also set default value on your enum class using ``__default__``
+attribute
```

Comment:
Well, `Enum` itself has only one place to set default i.e. via `__default__` attribute. Previously, one would need to override `.default()` method (still possible) and it was not obvious/expected that the first value is the default one. One could restore the old behaviour by subclassing Enum whenever it's needed:

```py
from django_enumfield import enum

class FirstDefaultEnum(enum.Enum):
    def default(cls):
        return list(cls)[0]
```

`EnumField` on the other hand may be independent enough to override the default just if it's needed. For example, the same enum may be used in various places and who knows maybe someone will need to override the default just in one of them. I believe one expects this to be possible in a field.